utils/addon_uuid.go

```diff
@@ -0,0 +1,52 @@
+package utils
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/go-utils/errors/v2"
+)
+
+func GetAddonUUIDFromType(ctx context.Context, app, addonUUIDorType string) (string, error) {
```

Comment:
nitpick: I found hard to read the variable `addonUUIDorType` maybe rename it to `addonTypeOrUUID`.
If you have a better name, use it :slightly_smiling_face: